CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.3.0 / 2022-03-28
+
+- [IMPROVEMENT] Use OkHttp 4.9.x in plugin dependency
```

Comment:
We should also mention the following customer-facing change https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/72